pytorch_lightning/loggers/wandb.py

```diff
@@ -89,13 +88,11 @@ class WandbLogger(LightningLoggerBase):
 
     def __init__(
         self,
-        name: Optional[str] = None,
```

Comment:
this will fail if a user uses positional arguments
pls, maintain a proper deprecation process...

Reply:
Ok I put back `name` and `project` at their previous positions.